saleor/csv/notifications.py

```diff
@@ -36,7 +36,7 @@ def send_export_download_link_notification(export_file: "ExportFile", data_type:
         **get_site_context(),
     }
 
-    manager = get_plugins_manager()
+    manager = get_plugins_manager(allow_replica=False)
```

Comment:
It looks like something that can use `allow_replica=True`. I see in the code, that this logic is triggered by celery task, so it should be okay to use here a replica DB.